components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -411,6 +413,17 @@ func Run(options ...RunOption) {
 		wg       sync.WaitGroup
 		shutdown = make(chan ShutdownReason, 1)
 	)
+	go func() {
```

Comment:
Do we really need it? Would GitpodWorkspaceTooManyRegularNotActiveMk2 trigger without it as well?

Reply:
I wonder what is the minimal change that we will cause GitpodWorkspaceTooManyRegularNotActiveMk2 for JB?

Reply:
We need it, there's no way user can stop workspaces in this state (except delete workspace but I assume they won't do it), it will 💸